src/main/java/io/github/jhipster/online/service/CiCdService.java

```diff
@@ -134,7 +134,7 @@ public void configureCiCd(User user, String organizationName, String projectName
             this.logsService.addLog(ciCdId, "Generation finished");
         } catch (Exception e) {
             this.logsService.addLog(ciCdId, "Error during generation: " + e.getMessage());
-            e.printStackTrace();
+            log.error(e.getMessage(), e);
```

Comment:
I would suggest `log.error("Generation Failed", e)` since `e.getMessage` will always be printed regardless when printing the stack-trace of the error.